package.json

```diff
@@ -42,7 +42,8 @@
     "clean": "rimraf docs/ coverage/",
     "lint": "eslint *.js test/*.js",
     "docs": "jsdoc --configure .jsdocrc *.js",
-    "test": "jest --coverage && yarn run lint && yarn run spellcheck"
+    "test": "jest --coverage && yarn run lint && yarn run spellcheck",
+    "update-snapshots": "jest -u"
```

Comment:
I can run `jest -u` in console by my `n` function https://github.com/ai/environment/blob/master/zshrc#L52-L61

Could you remove it?